src/pages/githubDiscussionPage.js

```diff
@@ -14,6 +14,8 @@ export default class GitHubDiscussionPage extends Page {
     return widgetZone.querySelector(".timeline-comment-actions") !== null;
   }
 
+  applyCSSPatch() {}
```

Comment:
Is this needed? The method it overrides is the same.

(understandable if this is just for legibility)

Reply:
Yeah it’s for legibility—so that all Page classes follow the same pattern